src/QueryDataTable.php

```diff
@@ -121,6 +121,16 @@ public function totalCount()
         return $this->totalRecords ? $this->totalRecords : $this->count();
     }
 
+    /**
+     * Count filtered items.
+     *
+     * @return int
+     */
+    public function filteredCount()
```

Comment:
Maybe we should promote this function to the abstract class since this will also be used when using Collection?